blog-post.html

```diff
@@ -1,99 +0,0 @@
-<!DOCTYPE html>
-<html>
-
-<head>
-  <title>Blog Post</title>
-  <meta charset="utf-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/css?family=Lora&display=swap" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/css?family=Arimo|Source+Sans+Pro&display=swap" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300&display=swap" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/css?family=Raleway&display=swap" rel="stylesheet">
-  <link rel="stylesheet" href="blog.css" />
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-</head>
-
-<body>
-  <div class="contain-blog">
-    <section id="header">
-      <div id="home-top">
-        <header>
-          <a href="index.html"><img src=beer-header.png /></a>

```

Comment:
To link to your index.html you can also set the `href = "/"` to send you back to the home page.